notes/common/prefix.go

```diff
@@ -55,7 +56,7 @@ func (t PRType) String() string {
 	case InfraPR:
 		return "infra"
 	default:
-		panic(fmt.Sprintf("unrecognized PR type %v", t))
+		panic("unrecognized PR type")
```

Comment:
if PRType is an int, can't we just
```suggestion
		panic(fmt.Sprintf("unrecognized PR type %d", int(t)))
```

Reply:
What was wrong with the previous code?

Reply:
> What was wrong with the previous code?

it is up in the PR description which I also missed at first

/lgtm

Reply:
Yeah, casting it to an int is better, thanks!